element-connector/src/main/java/org/eclipse/californium/elements/tcp/TlsServerConnector.java

```diff
@@ -58,6 +59,7 @@ public TlsServerConnector(KeyManagerFactory keyManagerFactory, InetSocketAddress
 
 	@Override protected void onNewChannelCreated(Channel ch) {
 		SSLEngine sslEngine = sslContext.createSSLEngine();
+		sslEngine.setWantClientAuth(true);
```

Comment:
Just to add: the SSLEngine supports "want" and "need". So I thought, with "want", the upper layer can do his checks.